README.md

```diff
@@ -8,7 +8,10 @@ In addition to handling proposed answers, this action will automatically close a
 
 ## Steps to enable this action in your repository
 
-1. A valid GitHub token is required to use this action. The GITHUB_TOKEN for your repository can be generated by GitHub during workflow execution when passed in as `${{ secrets.GITHUB_TOKEN }}`. Please check the permissions in Repository settings to grant write permissions to the token.
+1. A valid GitHub token is required to use this action. The GitHub token for your repository can be generated by GitHub during workflow execution when passed in as ${{ secrets.GITHUB_TOKEN }}. The GitHub token must have <b>discussions: write permissions</b>, these permissions can be granted in the workflow file.
```

Comment:
I think it will look best if we code highlight `${{ secrets.GITHUB_TOKEN }}`, `discussions: write`, and `Settings -> Actions -> General`. 

\`${{ secrets.GITHUB_TOKEN }}\` \`discussions: write\` \`Settings -> Actions -> General\`